keyboards/nifty_numpad/info.json

```diff
@@ -0,0 +1,153 @@
+{
+    "manufacturer": "Isaac Rex",
+    "keyboard_name": "Nifty Numpad",
+    "maintainer": "Acliad",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["GP17", "GP18", "GP15", "GP13", "GP14", "GP12"],
+        "rows": ["GP20", "GP21", "GP22", "GP23", "GP24", "GP25"]
+    },
+    "processor": "RP2040",
+    "rgb_matrix": {
+        "driver": "ws2812",
+        "max_brightness": 255,
+        "default_val": 10,
+        "default": {
+            "animation": "solid_color",
+            "hue": 127,
+            "speed": 100
+        },
+        "animations": {
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_sat": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "band_val": true,
+            "breathing": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "cycle_up_down": true,
+            "digital_rain": true,
+            "dual_beacon": true,
+            "gradient_up_down": true,
+            "jellybean_raindrops": true,
+            "multisplash": true,
+            "rainbow_beacon": true,
+            "rainbow_moving_chevron": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "solid_color": true,
```

Comment:
```suggestion
```

Reply:
@zvecr Both comments addressed and pushed. Thanks!